e2e-tests/gef/cypress/integration/unissued-facility-change-from-preview-AIN.spec.js

```diff
@@ -0,0 +1,258 @@
+import relative from './relativeURL';
+
+const CONSTANTS = require('../fixtures/constants');
+
+const { add, format } = require('date-fns');
+
+import {
+  MOCK_AIN_APPLICATION, MOCK_FACILITY_ONE, MOCK_FACILITY_TWO, MOCK_FACILITY_THREE, MOCK_FACILITY_FOUR, MOCK_USER_MAKER,
+} from '../fixtures/MOCKS/MOCK_DEALS';
+import applicationPreview from './pages/application-preview';
+import unissuedFacilityTable from './pages/unissued-facilities';
+import aboutFacilityUnissued from './pages/unissued-facilities-about-facility';
+import CREDENTIALS from '../fixtures/credentials.json';
+import applicationSubmission from './pages/application-submission';
+import statusBanner from './pages/application-status-banner';
+
+let dealId;
+let token;
+let facilityOneId;
+
+const unissuedFacilitiesArray = [
+  MOCK_FACILITY_ONE,
+  MOCK_FACILITY_THREE,
+  MOCK_FACILITY_FOUR,
+];
+
+const today = new Date();
+const oneMonth = add(today, { months: 1 });
+const oneMonthDay = format(oneMonth, 'dd');
+const oneMonthMonth = format(oneMonth, 'M');
+const oneMonthYear = format(oneMonth, 'yyyy');
+const twoMonths = add(today, { months: 2 });
+const twoMonthsDay = format(twoMonths, 'dd');
+const twoMonthsMonth = format(twoMonths, 'M');
+const twoMonthsYear = format(twoMonths, 'yyyy');
+const threeMonthsOneDay = add(today, { months: 3, days: 1 });
+const threeMonthsOneDayDay = format(threeMonthsOneDay, 'dd');
+const threeMonthsOneDayMonth = format(threeMonthsOneDay, 'M');
+const threeMonthsOneDayYear = format(threeMonthsOneDay, 'yyyy');
+
+context('Unissued Facilities AIN - change to issued from preview page', () => {
```

Comment:
Comparing to the other e2e test, should have the file name and context labelled as "single facility"? So it's clearer - because the other test is for multiple

Reply:
in this one, I am testing if I can change facilities from application preview (once one changed in unissued table as need to that to unlock).  What should I call it?

Reply:
if it's dependent on something being 'unlocked' would be best to mention this in the description/context? Up to you - I just didn't understand what the scenario is for this, compared to the other test